crates/precompile/src/blake2.rs

```diff
@@ -1,12 +1,21 @@
 use crate::{Error, Precompile, PrecompileResult, PrecompileWithAddress};
 use revm_primitives::Bytes;
 
+#[cfg(feature = "scroll")]
+use revm_primitives::PrecompileError;
+
 const F_ROUND: u64 = 1;
 const INPUT_LENGTH: usize = 213;
 
 pub const FUN: PrecompileWithAddress =
     PrecompileWithAddress(crate::u64_to_address(9), Precompile::Standard(run));
 
+#[cfg(feature = "scroll")]
+pub const BERNOULLI: PrecompileWithAddress = PrecompileWithAddress(
+    crate::u64_to_address(9),
+    Precompile::Standard(|_input: &Bytes, _gas_limit: u64| Err(PrecompileError::OutOfGas)),
```

Comment:
not sure whether use "PrecompileError::OutOfGas" is good.  let us recheck later.   Err(PrecompileError::NotImplemented)?

Reply:
done

Reply:
Ok